homeassistant/components/esphome/ffmpeg_proxy.py

```diff
@@ -147,12 +158,12 @@ async def prepare(self, request: BaseRequest) -> AbstractStreamWriter | None:
             stderr=asyncio.subprocess.PIPE,
         )
 
+        # Only one conversion process per device is allowed
+        self.convert_info.proc = proc
```

Comment:
I think that you should also check/kill existing process here, to ensure that a user cannot open this url a 1000 times and get 1000 processes

Reply:
The HTTP view will do this already.